frontend/src/app/organization/organization-create/organization-create.component.ts

```diff
@@ -43,48 +46,22 @@ export class OrganizationCreateComponent implements OnInit, OnDestroy {
        'description': this.fb.control(null, Validators.required),
       });
 
-    this.orgSubscription = this.organizationService.organization$
-      .pipe(skip(1), take(1))
-      .subscribe((organization: Organization) => this.createForm.patchValue(organization));
-
-    this.activatedRoute.params
-    .subscribe(params => {
-        if (params.organizationId) {
-          this.inEditMode = true;
-          this.id = params.organizationId;
-          this.organizationService.getOrganization(params.organizationId);
-        } else {
-          this.orgSubscription.unsubscribe();
-        }
-    });
-    // this.activatedRoute.params
-    //   .pipe(
-    //     tap(params => {
-    //       if (params.organizationId) {
-    //         this.inEditMode = true;
-    //         this.id = params.organizationId;
-    //         this.organizationService.getOrganization(params.organizationId);
-    //       }
-    //     }),
-    //     switchMap(() => this.organizationService.organization$)
-    //   )
-    //   .subscribe(
-    //     (organization: Organization) => this.createForm.patchValue(organization)
-    //   );
+    this.organization$ = this.activatedRoute.params.pipe(
+      map(params => params.organizationId),
+      filter(organizationId => !!organizationId),
+      switchMap(organizationId => this.organizationService.getOrganization(organizationId)),
+      tap(organization => this.createForm.patchValue(organization))
+    );
 
     this.createSubscription = this.organizationService.createOrEditOrganization$
       .subscribe(
         (response) => {
           if (response.type !== 'error') {
-            const newUser = Object.assign({}, this.user);
-            if (this.inEditMode) {
-              newUser.organizations[newUser.organizations.findIndex(
-                organization => { return organization.id === this.id}
-                )] = response.data;
+            if (response.type === 'edit') {
+              this.userService.updateOrganization(response.data).subscribe();
```

Comment:
Sorry if you feel offfended.

I reply fast as I'm notified on some PRs, otherwise I would probably forget about it if I kept it for later. I think part of the misunderstanding comes from the fact we discuss some things locally which for obvious reasons you can't participate.

I think you could take a look how it's perceived here. So overall it happens that there is some issue with an ongoing PR, we discuss briefly on the meetup what we can do, usually the code is not finished, then you add some numerous comments that these changes are actually wrong or 'broken', even if not yet done. So yes, it's been a bit annoying for me as well at times and sorry for that.

The only reason I ever touched this PR is because Julia asked for some help during the meetup and there have been some issues with subscriptions, which we did not share with you. That most likely have been a mistake.
Sorry if I apparently used the tone you considered spiteful, even though I tried to keep it content-related.

Overall, I'm **trying** to help just like you. I think it's best if you actually joined us at at some point in Poznań (what Magul suggested) and grabbed some beer, which will help us to communicate remotely better.